.travis.yml

```diff
@@ -4,7 +4,6 @@ dist: trusty
 sudo: required
 language: java
 jdk:
-  - openjdk7
   - oraclejdk8
   - oraclejdk9
```

Comment:
Add `oraclejdk10` and `oraclejdk11` here would be good. We can allow failures for `oraclejdk11` by adding the following configuration:

```yaml
matrix:
  allow_failures:
    - jdk: oraclejdk11
```